Gruntfile.js

```diff
@@ -147,6 +147,13 @@ module.exports = function(grunt) {
   var configJSON = {
     pkg: grunt.file.readJSON("package.json"),
     bump: bumpJSON,
+    umd: {
+      all: {
+        src: "plottable.js", 
+        objectToExport: "Plottable",
+        amdModuleId: "Plottable",
```

Comment:
What happens if we don't set `objectToExport` and `amdModuleId`?  I'm reading the documentation on https://github.com/bebraw/grunt-umd and I'm unsure what's the advantages / disadvantages of setting this directly